build/flutter_app.gni

```diff
@@ -132,6 +133,8 @@ template("flutter_app") {
     ]
     if (defined(invoker.manifest)) {
       inputs += [ rebase_path(invoker.manifest) ]
+    } else {
+      inputs += [ rebase_path(flutter_tools_asset_manifest)]
     }
```

Comment:
We should set a local variable called `manifest` that is either `invoker.manifest` (if that is defined) or `flutter_tools_asset_manifest`.  Then we can add it to `inputs` and construct the command line below without conditionals.